lib/v2/screens/dashboard/transactions/components/transaction_info_card.dart

```diff
@@ -20,74 +22,79 @@ class TransactionInfoCard extends StatelessWidget {
+                      decoration: const BoxDecoration(
+                        shape: BoxShape.circle,
+                        color: AppColors.lightGreen2,
                       ),
-                      const SizedBox(height: 10),
-                      Row(
-                        children: <Widget>[
-                          Expanded(
-                              child: Text(timesTampToTimeAgo(timestamp),
-                                  style: Theme.of(context).textTheme.subtitle2OpacityEmphasis)),
-                          Text(amount.symbolFromAmount, style: Theme.of(context).textTheme.subtitle2OpacityEmphasis)
-                        ],
+                    ),
+                    Expanded(
+                      child: Container(
+                        padding: const EdgeInsets.only(left: 10),
+                        child: Column(
+                          crossAxisAlignment: CrossAxisAlignment.start,
+                          children: <Widget>[
+                            Row(
+                              children: <Widget>[
+                                Expanded(
+                                    child: Text(
+                                  state.displayName,
+                                  style: Theme.of(context).textTheme.button,
+                                  overflow: TextOverflow.ellipsis,
+                                  maxLines: 1,
+                                )),
+                                const SizedBox(width: 40),
+                                Text(amount.seedsFormatted, style: Theme.of(context).textTheme.button),
+                                const SizedBox(width: 4),
+                                SvgPicture.asset(
+                                    incoming
+                                        ? 'assets/images/wallet/arrow_up.svg'
+                                        : 'assets/images/wallet/arrow_down.svg',
+                                    height: 16)
+                              ],
+                            ),
+                            const SizedBox(height: 10),
+                            Row(
+                              children: <Widget>[
+                                Expanded(
+                                    child: Text(timesTampToTimeAgo(timestamp),
+                                        style: Theme.of(context).textTheme.subtitle2OpacityEmphasis)),
+                                Text(amount.symbolFromAmount,
+                                    style: Theme.of(context).textTheme.subtitle2OpacityEmphasis)
+                              ],
+                            ),
+                          ],
+                        ),
                       ),
-                    ],
-                  ),
+                    )
+                  ],
                 ),
-              )
-            ],
-          ),
-        ),
-      ),
-    );
+              ),
+            ),
+          );
+        }));
```

Comment:
line 98 commas

Reply:
What do you mean? Not sure what the rule here is?